.github/workflows/tests.yaml

```diff
@@ -21,10 +18,10 @@ jobs:
       - name: Check out the repository
         uses: actions/checkout@v4.2.2
 
-      - name: Set up Python ${{ env.DEFAULT_PYTHON }}
-        uses: actions/setup-python@v2.2.1
+      - name: Set up Python
+        uses: actions/setup-python@v5.4.0
         with:
-          python-version: ${{ env.DEFAULT_PYTHON }}
+          python-version-file: ".python-version"
```

Comment:
This was introduced in [v4.0](https://github.com/actions/setup-python/releases/tag/v4.0.0)